service-commands/src/setup.js

```diff
@@ -711,29 +728,51 @@ const allUp = async ({
   await runInParallel(ipfsAndContractsCommands, options)
 
   // Run sequential ops
-  await runInSequence(sequential1, options)
+  await runInSequence(prereqs, options)
 
   if (parallel) {
+    const startProv = Date.now()
+    console.log('Provisioning services in parallel'.info)
     await Promise.all(
-      discoveryNodesCommands.map(commandGroup =>
+      nodeUpCommands.map(commandGroup => runInSequence(commandGroup, options))
+    )
+    console.log(
+      `Services provisioned in ${Math.abs((Date.now() - startProv) / 1000.0)}s`
+        .info
+    )
+    const startHealth = Date.now()
+    console.log('Health checking services'.info)
+    await Promise.all(
```

Comment:
These healthchecks could probably be run sequentially - by the time one is done, they're usually all pretty much done anyway. But I was going for _hyperspeed_